selfdrive/controls/controlsd.py

```diff
@@ -218,6 +219,10 @@ def update_events(self, CS):
       self.events.add(EventName.controlsInitializing)
       return
 
+    # Block resume if cruise never previously enabled
+    if not self.CP.pcmCruise and self.v_cruise_kph == V_CRUISE_INITIAL:
+      self.events.add(EventName.resumeBlocked)
+
```

Comment:
@adeebshihadeh should this go in SIMULATION or should we get a new process replay segment where set is used first?

Reply:
think it makes sense to set it here https://github.com/commaai/openpilot/blob/3fc001c945bb5e9092050bac1e55267dab63b9eb/selfdrive/controls/controlsd.py#L422

Reply:
The route doesn't start engaged though, they enable with resume 10 seconds in. I don't want to skip the state machine by checking this all the time